app/graphql/mutations/integrations/sync_invoice.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+module Mutations
+  module Integrations
+    class SyncInvoice < BaseMutation
+      include AuthenticableApiUser
+      include RequiredOrganization
+
+      REQUIRED_PERMISSION = 'organization:integrations:update'
+
+      graphql_name 'SyncIntegrationInvoice'
+      description 'Sync integration invoice'
+
+      input_object_class Types::Integrations::SyncInvoiceInput
+
+      field :invoice_id, ID, null: true
+
+      def resolve(**args)
+        invoice = current_organization.invoices.find_by(id: args[:invoice_id])
+
+        ::Integrations::Aggregator::Invoices::CreateJob.perform_later(invoice:)
```

Comment:
I'm wondering if we could introduce a `call_async` logic into some services that just need to call an external service:

```ruby
module Integrations
  module Aggregator
    module Invoices
      class CreateService

# ...
        def call_async
          return result.not_found_failure!(resource: 'invoice') unless invoice

          ::Integrations::Aggregator::Invoices::CreateJob.perform_later(invoice:)
          result.invoice_id = invoice.id
          result
        end
```

It brings 3 advantages:
- keeping the presence check for the invoice and render "NOT FOUND" if applicable
- building a proper result inside the service and not in the mutation
- keeping classic `result.success? ? result.invoice_id : result_error(result)` result handing in the mutation

In the future, if we see more of this, we could even implement the pattern at service level